README.md

```diff
@@ -0,0 +1,42 @@
+# keeper
+
+## Setup
+
+This is how to get keeper running on your machine so that you're able to work on it.
+
+First you have to login to clubhouse and generate an API key. You can do this from your account settings -> API key. Save this token somewhere and add it to a new environment variable called `CLUBHOUSE_API_KEY`.
```

Comment:
could specify that environment variables should be added to `/server/.env`

Reply:
So I haven't actually done this and don't know how it works? Do you want to add it or explain it and I can write it to the readme?

Reply:
So you can create a new file called `.env` in the `server` directory and add the line

```
CLUBHOUSE_API_KEY=abcd-efgh-ijkl
```